fluent_contents/plugins/twitterfeed/models.py

```diff
@@ -9,14 +9,21 @@ class TwitterRecentEntriesItem(ContentItem):
     """
     Content item to display recent entries of a twitter user.
     """
-    title = models.CharField(_('Title'), max_length=200, blank=True, help_text=_('You may use Twitter markup here, such as a #hashtag or @username.'))
+    title = models.CharField(_('Title'), max_length=200, blank=True,
+        help_text=_('You may use Twitter markup here, such as a #hashtag or @username.'))
```

Comment:
I guess this no longer applies now that `urlize_twitter` is removed.
Would be nice to keep that in though.